src/test/java/org/dinosaur/foodbowl/domain/comment/repository/CommentRepositoryTest.java

```diff
@@ -14,6 +16,28 @@ class CommentRepositoryTest extends RepositoryTest {
     @Autowired
     private CommentRepository commentRepository;
 
+    @Test
+    @DisplayName("댓글을 저장한다.")
+    void createComment() {
+        Post post = postTestSupport.postBuilder().build();
+        Member member = memberTestSupport.memberBuilder().build();
+        Comment comment = Comment.builder()
+                .post(post)
+                .member(member)
+                .parent(null)
+                .message("정답은 없다. 자신만의 기준을 가지면 된다.")
+                .build();
+
+        Comment savedComment = commentRepository.save(comment);
+
+        Assertions.assertAll(
+                () -> assertThat(savedComment.getMember()).isEqualTo(comment.getMember()),
```

Comment:
```suggestion
                () -> assertThat(savedComment.getId()).isNotNull(),
                () -> assertThat(savedComment.getMember()).isEqualTo(comment.getMember()),
```
반환된 댓글은 id가 존재한다 검증 어떤가요?

Reply:
추가했습니당 ㅎㅎ